### PR TITLE
[ADD] account_banking_make_deposit

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

File: account_banking_make_deposit/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,685 @@
...
+msgid "Check Number"
+msgstr ""
+
+#. module: account_banking_make_deposit
+#: model:ir.model.fields,field_description:account_banking_make_deposit.field_deposit_items_line__company_id
+#: model:ir.model.fields,field_description:account_banking_make_deposit.field_deposit_ticket__company_id
+#: model:ir.model.fields,field_description:account_banking_make_deposit.field_deposit_ticket_line__company_id
+#: model_terms:ir.ui.view,arch_db:account_banking_make_deposit.deposit_ticket_report
+msgid "Company"
+msgstr "Cancelar"
```
Comment (max3903) [inline account_banking_make_deposit/i18n/es.po]:
```suggestion
msgstr "Compañía"
```

**Review by nelsonramirezs — APPROVED** [review]

**Review by nelsonramirezs — APPROVED** [review]

**Review by bodedra — APPROVED** [review]
Code review. :+1:

**Comment by dreispt** [comment]
@max3903 This seems to be an old module authored by someone else, but we don't have a commit history. Can you please detail it's source?

**Pull request merged**